FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Statistics.java

```diff
@@ -15,17 +15,19 @@ public enum Statistics {
     CUMULATIVE,
     MAXAVERAGEDURATION,
     MAXACCUMDURATION,
-    COMPUTABLE;
+    PLOTTINGPOSITION,
+    COMPUTABLE,
+    UNDEFINED;
 
-    public static String pack(EnumSet<Statistics> set){ //returns a list of the statistics in a string format
-        String ret = "";
+    public static String pack(Set<Statistics> set){ //returns a list of the statistics in a string format
+        StringBuilder ret = new StringBuilder();
         for (Statistics s : Statistics.values()){
             if (set.contains(s)){
-                ret += s.name() + ",";
+                ret.append(s.name()).append(",");
             }
         }
-        ret = ret.substring(0,ret.length()-1); // removes the last comma
-        return ret;
+        ret = new StringBuilder(ret.substring(0, ret.length() - 1)); // removes the last comma
```

Comment:
string builders are ideal for multiple appends, this use of string builder doesnt make alot of sense.